crypto/bls12381_utils.go

```diff
@@ -141,6 +141,14 @@ func writePointG2(dest []byte, a *pointG2) {
 	)
 }
 
+// writePointG1 writes a G1 point in a slice of bytes
+func writePointG1(dest []byte, a *pointG1) {
```

Comment:
Are there any requirements on the slice length or capacity? If so, would be great to add them to the goDoc.